### DISCUSSION TITLE
eez studio support (open source squareline studio alternative)

### DISCUSSION THREAD

**Discussion by ClemensEy** [sukesh-ak/ESP32-TUX#55] [Ideas]
please consider supporting eez studio including the flowchart coding functionality. Squareline studio seems obsolete since it is no longer connected to lvgl and it is expensive.

that would be great!

**Comment by sukesh-ak** [comment]
LVGL is working on their own designer/editor as well. Waiting for that.

**Comment by ClemensEy** [comment]
Thanks for the update
